File: jcarrollonlinev4.frontend/src/models/ForumThreadEntries/ThreadEntryDetailsModel.ts
import { ForaGetModel } from "../Fora/ForaGetModel";
import { ThreadEntriesModelBase } from "./ThreadEntriesModelBase";
import { ThreadEntryDetailItemsModel } from "./ThreadEntryDetailItemsModel";

    export interface ThreadEntryDetailsModel extends ThreadEntriesModelBase
    {
      ForumThreadEntryDetailItems: ThreadEntryDetailItemsModel;

      ForaDetailsModel: ForaGetModel;
      Replies: number;
    }

File: jcarrollonlinev4.backend/Models/ForumThreadEntries/ThreadEntryDetailsModel.cs
using jcarrollonlinev4.backend.Models.Fora;

namespace jcarrollonlinev4.backend.Models.ForumThreadEntries
{
    public class ThreadEntryDetailsModel : ThreadEntriesModelBase
    {
        private ThreadEntryDetailItemsModel _forumThreadEntryDetailItems = new ThreadEntryDetailItemsModel();
        private ForaDetailsModel _foraDetailsModel = new ForaDetailsModel();

        public ThreadEntryDetailItemsModel ForumThreadEntryDetailItems => _forumThreadEntryDetailItems;

        public ForaDetailsModel Forum => _foraDetailsModel;
        public int Replies { get; set; }
    }
}
